django/tasks/base.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.module_loading import import_string
+
+from .exceptions import InvalidTaskBackendError
+
+DEFAULT_TASK_ALIAS = 'default'
+
+# backends cache to make sure we don't create one per connection
+# all the backends should be thread-safe
+backends = {}
+
+
+def _create_task_backend(alias):
+    try:
+        conf = settings.QUEUES[alias]
+    except KeyError:
+        if alias != DEFAULT_TASK_ALIAS:
+            return _create_task_backend(DEFAULT_TASK_ALIAS)
+        raise ImproperlyConfigured("%s is not defined in QUEUES" % alias)
```

Comment:
So in the current implementation, this error can only ever possibly be raised when `alias == 'default'`. In which case there's no real point in interpolating `alias`.